pxr/usdImaging/lib/usdviewq/appController.py

```diff
@@ -2355,6 +2361,9 @@ def _reopenStage(self):
         QtWidgets.QApplication.setOverrideCursor(QtCore.Qt.BusyCursor)
 
         try:
+            # Pause the stage view while we update
+            self._stageView.setUpdatesEnabled(False)
+
```

Comment:
Unrelated change, but I understand why you bundled it 8-)

Reply:
:)